src/css/css-pass.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {ParsedCssDocument} from 'polymer-analyzer/lib/css/css-document';
+import {UpgradePass} from '../upgrade-pass';
+
+export abstract class CssUpgradePass extends UpgradePass<ParsedCssDocument> {
```

Comment:
When I see this class I wonder how to use it. It's abstract, so what needs to be filled in by a subclass? It has a field, what's it for? My comment probably landed here first rather than UpgradePass because alphabets.